### PR TITLE
wireless bridge page

### PR DESCRIPTION
**Pull request by jlpoltrack**
Initial attempt, the sketch is very well documented with comments so tried to keep it pretty simple.

### CODE REVIEW

**Comment by olliw42** [comment]
fantastic.
it lgtm already :)

I wonder where to put it on the mnain docu page, the existing chapeters don't seem to fit well. Maybe Setups. Or you're thinking about openeing an new chapter for it?

**Comment by jlpoltrack** [comment]
> I wonder where to put it on the mnain docu page, the existing chapeters don't seem to fit well. Maybe Setups. Or you're thinking about openeing an new chapter for it?

I feel that 'General' may be better than 'Setups' but don't have strong opinion here.

**Comment by olliw42** [comment]
I do have an opinion here however :) The General section appears to hold topics which are (1) very specific to the mLRS modules and (2) important to know to get started. This view also goes along with that it is the first chapter on the page. Wireless-bridge is not the same category, one in fact can get well along without even nowing about it. This would place it rather in between Hardware and Firmware/Build Notes ... if there would be a section ... 
In short, I think it's given a way too prominent place, distracting users. 
IMHO. :)

**Comment by jlpoltrack** [comment]
> an OLED page would probably fall into the same category as wireless bridge

Was thinking this as well - how about an 'Optional Hardware' chapter?

**Comment by olliw42** [comment]
it would work great for the two cases ... but I'm not sure what would come else. E.g. mavlinkX? In some sense also Relay might fit better into here. Or Dualband (if it ever comes). Or what the heck. This makes think rather interms of an agnoistic container, but my mind just can't come up with a title ... like Miscellaneous or Bits And Bobs but just more specific and better LOL
I don't know. If nothing strikes you, I think 'Optional Hardware' will work great for now.
